### DISCUSSION TITLE
Have a look at non-reversible samplers

### DISCUSSION THREAD

**Discussion by MichaelClerx** [pints-team/pints#1536] [New methods]
Bouncy particle sampler: https://doi.org/10.1080/01621459.2017.1294075

Zig-zag: https://arxiv.org/abs/1607.03188

**Comment by MichaelClerx** [comment]
@ben18785 do you know these? Think they're worth keeping on the possibly-to-do-list?

**Comment by ben18785** [comment]
Yes please let's keep open
